### DISCUSSION TITLE
这里有办法注销账户吗

### DISCUSSION THREAD

**Discussion by ghost** [BakaXL-Support/BakaXL-QA-docs#138] [使用时的疑问] [answered]
我的邮箱需要注销但是我的BakaXL账户仍然在用这个邮箱，有办法注销bakaxl账户吗

**Comment by TT702** [comment, marked as answer]
你可以注销，也可以换到其他邮箱上。
如果你要换，可以用现在账户用的邮箱发邮件到TT702@outlook.com，里面写上你要改的邮箱

**Reply by ghost** [reply]
那我要注销账号还是发到这个邮箱吗

**Reply by TT702** [reply]
对的，但是注销后你的好友信息、赞助者身份计划都会消失哦。

**Reply by ghost** [reply]
好的谢谢,已经发过去了

**Reply by ghost** [reply]
> 你可以注销，也可以换到其他邮箱上。 如果你要换，可以用现在账户用的邮箱发邮件到[TT702@outlook.com](mailto:TT702@outlook.com)，里面写上你要改的邮箱

这个注销的流程大约要多久
